### ISSUE TITLE
Any suggestions to help with the build time for docs.rackspace.com

### ISSUE THREAD

**Issue by steveortiz** [rackerlabs/standard-usage-schemas#529]
We having some trouble with the time it takes maven to build docs for standard-usage-schemas.  It takes a relatively long time build (about 10 minutes on my laptop), and a majority of the time the commits triggering the build have no documentation related changes.  So, I was hoping you could help us out with a couple things.

1) Is there a way to make the docs build faster? (maybe skipping the TESTS phase)  Here is the command we run: `mvn clean generate-sources`

2) If we can't make it build faster, is there anything we could do to make it build less often?  Perhaps creating a branch "publish-to-docs" that is only used to trigger the build process.  We could setup our tools to track that branch instead of master, and we'd merge from master to that branch when we wanted to kick off the docs build process?

Making it build faster is the preferred method (IMO).  Thanks in advance for your help.

**Comment by shintasmith** [comment]
@steveortiz, yes, we are aware that the build takes long. This is due to the fact that api-checker/wadl-tools takes a while to load and compile the WADL, since our WADL is quite big. And yes, this happens when you are running tests.

If you are executing this command ```mvn clean generate-sources process-sources -Dwebhelp.war=1```, this should already skipped the tests, since the Maven test phase is after the generate-sources and process-sources phases. 

Can you send me a link to your Jenkins job? Perhaps the job is not configured properly?

**Comment by j12y** [comment]
Hi @shintasmith 
We run a process that is effectively just running this command:

```
$ /usr/local/apache-maven/bin/mvn -f pom.xml clean -Dgenerate.docs generate-sources process-sources -Dwebhelp.war=1 -Dmaven.test.skip=true -Dcobertura.skip=true

[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 34:46 min
[INFO] Finished at: 2015-10-08T19:53:43+00:00
[INFO] Final Memory: 622M/2216M
[INFO] ------------------------------------------------------------------------
```
